### PR TITLE
cdsclient 3.84 (new formula)

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/cdsclient.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Cdsclient < Formula
+  desc "Tools for querying CDS databases"
+  homepage "https://cdsarc.u-strasbg.fr/doc/cdsclient.html"
+  url "http://cdsarc.u-strasbg.fr/ftp/pub/sw/cdsclient-3.84.tar.gz"
+  sha256 "09eb633011461b9261b923e1d0db69d3591d376b447f316eb1994aaea8919700"
+  license "GPL-3.0-only"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--libdir=#{lib}"
```
Comment (chenrui333) [inline Formula/cdsclient.rb:9]: maybe this?
```suggestion
    system "./configure", *std_configure_args
```

Comment (alebcay) [reply Formula/cdsclient.rb:9]:
```
configure: WARNING: unrecognized options: --disable-debug, --disable-dependency-tracking, --disable-silent-rules
```

Comment (chenrui333) [reply Formula/cdsclient.rb:9]: 👍

**Review by chenrui333 — APPROVED** [review]

**Review by iMichka — APPROVED** [review]

File: Formula/cdsclient.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class Cdsclient < Formula
+  desc "Tools for querying CDS databases"
```
Comment (carlocab) [inline Formula/cdsclient.rb:2]: Maybe we can spell out what CDS is in this context?

**Pull request closed without merging**